console_backend/Cargo.toml

```diff
@@ -20,7 +20,7 @@ directories = "5"
 crossbeam = "0.8"
 rand = "0.8.5"
 parking_lot = "0.12.1"
-sbp-settings = "0.6.16"
+sbp-settings = {git = "https://github.com/swift-nav/libsettings-rs.git", branch = "adrian/update-baudrate-enum"}
```

Comment:
this is to be changed after https://github.com/swift-nav/libsettings-rs/pull/82 gets merged in and released.